Description: disable running of fix_permissions build-time
 It's not compatible with fakeroot and we're not using the dir anyway.
Author: Thijs Kinkhorst <thijs@debian.org>
Forwarded: not-needed
Last-Update: 2020-04-28

--- domjudge-7.3.0.orig/Makefile
+++ domjudge-7.3.0/Makefile
@@ -124,7 +124,7 @@ install-domserver-l:
 		$(INSTALL_WEBSITE) -m 0775 -d $(DESTDIR)$(domserver_webappdir)/var/$$d ; \
 	done
 # Make sure that domjudge user and webserver group can write to var/{cache,log}:
-	DESTDIR=$(DESTDIR) $(domserver_bindir)/fix_permissions
+#	DESTDIR=$(DESTDIR) $(domserver_bindir)/fix_permissions
 # Special case create tmpdir here, only when FHS not enabled:
 ifneq "$(FHS_ENABLED)" "yes"
 	-$(INSTALL_WEBSITE) -m 0770 -d $(DESTDIR)$(domserver_tmpdir)
